db/migrate/20190617100508_add_limit_exceeded_error_to_proxies.rb

```diff
@@ -0,0 +1,7 @@
+class AddLimitExceededErrorToProxies < ActiveRecord::Migration
+  def change
+    safety_assured { add_column :proxies, :error_headers_limits_exceeded, :string, default: 'text/plain; charset=us-ascii' }
```

Comment:
`safety_assured` is only needed when you are sure everything was assured before that it will not be dangerous anymore

Reply:
we should have this conversation in 2 threads 😕 
let's talk about it either here or in https://github.com/3scale/porta/pull/906#discussion_r295167579